src/functions.php

```diff
@@ -0,0 +1,62 @@
+<?php
+require_once 'connect.php';
+
+$db = connectToDatabase();
+
+function pullPlantsFromDatabase(PDO $db): array
+{
+    $query = $db->prepare('SELECT `plants`.`name`, `plants`.`growth_rate_id`, `plants`.`watering_needs`, `plants`.`pet_friendliness`, `plants`.`fertilising_needs`, `plants`.`photo`, `growth_rates`.`rate` FROM `plants` JOIN `growth_rates` ON `plants`.`growth_rate_id` = `growth_rates`.`id`');
+
+    $result = $query->execute();
+
+    if ($result) {
+        $plants = $query->fetchALL();
+        return $plants;
+    } else {
+        return [];
+    }
+}
+
+
+function displayPlants(array $plants): string
+{
+    $result = '';
+    foreach ($plants as $plant){
+
+    if ($plant['pet_friendliness'] === 1){
+        $petFriendly = 'Considered pet friendly';
+    }else{
```

Comment:
```suggestion
    } else {
```

Let the `else` breathe.